radiomanager/radiomanager/__init__.py

```diff
@@ -34,14 +23,47 @@ class TESTING_CONFIG:
     SQLALCHEMY_DATABASE_URI = "sqlite://"
 
 
-def create_app(config_obj=PRODUCTION_CONFIG):
+def create_app(config_obj=TESTING_CONFIG):
     from .api import setup_routing
     app = Flask(__name__)
     app.config.from_object(config_obj)
     setup_routing(FlaskExtensions.api)
     with app.app_context():
         FlaskExtensions.api.init_app(app)
         FlaskExtensions.db.init_app(app)
+        FlaskExtensions.db.create_all()
```

Comment:
Running `create_all` on every startup isn't valid if the database is persistent.

Reply:
removed.